### PR TITLE
Add support for Django 3.2

### PR DESCRIPTION
**Pull request by francoisfreitag**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ ChangeLog
 *New:*
 
     - :issue:`366`: Add :class:`factory.django.Password` to generate Django :class:`~django.contrib.auth.models.User`
-      passwords.
```
Comment (sarahboyce) [inline docs/changelog.rst:12]: I think `passwords.` has been accidentally trimmed off the previous change log?

File: tox.ini [hunk 2]
```diff
@@ -17,8 +18,9 @@ deps =
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django32: Django~=3.2.0
```
Comment (sarahboyce) [inline tox.ini:21]:
```suggestion
    django32: Django>=3.2,<3.3
```
To be similar to the 2.2 example? Might cover any future patch releases

**Review by sarahboyce — COMMENTED** [review]
Some minor suggestions but otherwise looks good to me 👍

File: docs/changelog.rst (hunk 1 above)
Comment (francoisfreitag) [reply docs/changelog.rst:12]: Thanks, I’m not sure what happened, but I certainly did not mean to remove this word.

File: tox.ini (hunk 2 above)
Comment (francoisfreitag) [reply tox.ini:21]: The [`~=`](https://www.python.org/dev/peps/pep-0440/#compatible-release) operator means compatible release. It is equivalent to `>=3.2,<3.3`.
Not feeling strongly about specifying one way or another.

**Comment by francoisfreitag** [comment]
Thanks for the review!

File: tox.ini (hunk 2 above)
Comment (rbarrois) [reply tox.ini:21]: I tend to side with @sarahboyce here, the `>=3.2,<3.3` is easier to understand for a reader who doesn't spend their time in pip requirement files :D

File: tox.ini [hunk 3]
```diff
@@ -5,6 +5,7 @@ envlist =
     py{36,37,38,39,py3}-django22-alchemy-mongoengine
     py{36,37,38,39,py3}-django30-alchemy-mongoengine
     py{36,37,38,39,py3}-django31-alchemy-mongoengine
+    py{36,37,38,39,py3}-django32-alchemy-mongoengine
```
Comment (rbarrois) [inline tox.ini:8]: @francoisfreitag Have you checked the support policy for Django 3.2 w.r.t Python versions?

Comment (francoisfreitag) [reply tox.ini:8]: Yup: https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django.

**Review by rbarrois — APPROVED** [review]

**Comment by francoisfreitag** [comment]
Changes: Add a changelog entry.

**Pull request merged**